src/pages/home/sidebar/OptionRowTitle/index.js

```diff
@@ -0,0 +1,134 @@
+import _ from 'underscore';
+import React, {Fragment, PureComponent} from 'react';
+import {
+    Text,
+    View,
+} from 'react-native';
+import {propTypes, defaultProps} from './OptionRowTitleProps';
+import styles from '../../../../styles/styles';
+import Tooltip from '../../../../components/Tooltip';
+import hasEllipsis from '../../../../libs/hasEllipsis';
+
+class OptionRowTitle extends PureComponent {
+    constructor(props) {
+        super(props);
+        this.ref = null;
+        this.cRefs = [];
+        this.setContainerRef = (ref) => {
+            this.ref = ref;
+        };
+        this.setOptionChildRef = index => (ref) => {
```

Comment:
The double arrow functions here makes this difficult to understand what's going on. Please refactor this to be more clear or provide code comments to explain how this works.